Article/Article.js

```diff
@@ -3,17 +3,24 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
```

Comment:
Just another way of writing this if you're curious:
`this.expandButton = domElement.getElementsByClassName("expandButton")[0];